yellowbrick/text/dispersion.py

```diff
@@ -289,26 +321,32 @@ def dispersion(words, corpus, y=None, ax=None, colors=None, colormap=None,
         are positioned at the end of each document.
 
     ignore_case : boolean, default: False
-	Specify whether input  will be case-sensitive.
+        Specify whether input  will be case-sensitive.
 
     kwargs : dict
         Pass any additional keyword arguments to the super class.
 
     Returns
     -------
-    ax: matplotlib axes
-        Returns the axes that the plot was drawn on
+    viz: DispersionPlot
+        Returns the fitted, finalized visualizer
     """
 
     # Instantiate the visualizer
     visualizer = DispersionPlot(
-        words, ax=ax, colors=colors, colormap=colormap,
-        ignore_case=ignore_case, labels=labels,
-        annotate_docs=annotate_docs, **kwargs
+        words,
+        ax=ax,
+        colors=colors,
+        colormap=colormap,
+        ignore_case=ignore_case,
+        labels=labels,
+        annotate_docs=annotate_docs,
+        **kwargs
     )
 
     # Fit and transform the visualizer (calls draw)
     visualizer.fit(corpus, y, **kwargs)
+    visualizer.finalize()
 
-    # Return the axes object on the visualizer
-    return visualizer.ax
+    # Return the visualizer objec
```

Comment:
Not worth changing and waiting for CI to run again - but just wanted to prove that I was doing a thorough review ;-)

Reply:
good catch!